modules/uoscore-uedhoc/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if (CONFIG_UOSCORE OR CONFIG_UEDHOC)
+
+  set(MODULE_SRC_DIR ${ZEPHYR_CURRENT_MODULE_DIR}/src)
+  set(MODULE_INCLUDE_DIR ${ZEPHYR_CURRENT_MODULE_DIR}/inc)
```

Comment:
if those are global variables, please give them some more specific names. `MODULE_SRC_DIR` and `MODULE_INCLUDE_DIR` are too generic.

If those are local variables, please use lower casing.

Reply:
Renamed to `UOSCORE_UEDHOC_SRC_DIR`/`UOSCORE_UEDHOC_INCLUDE_DIR`.